extractor/filesystem/secrets/convert/convert.go

```diff
@@ -95,9 +101,13 @@ var _ filesystem.Extractor = &detectorWrapper{}
 func SetupVelesExtractors(extractors []filesystem.Extractor) ([]filesystem.Extractor, error) {
 	result := make([]filesystem.Extractor, 0, len(extractors))
 	detectors := []veles.Detector{}
+
 	for _, e := range extractors {
-		if d, ok := e.(veles.Detector); ok {
+		if d, isDetector := e.(veles.Detector); isDetector {
 			detectors = append(detectors, d)
+			if _, isExtractorPlaceHolder := e.(extractorPlaceHolder); !isExtractorPlaceHolder {
```

Comment:
Can you add a test for this? e.g. to scalibr_test.go where we have other validator conversion tests like `Veles_secret_detector_with_validation`